README.md

```diff
@@ -50,20 +50,41 @@ You must pass a `src` property with the link of the image:
 
 That image will be loaded as soon as the image enters the viewport.
 
-## Progressive loading
+## Responsive Images
+
+Using the `srcset` prop in combinations with the `sizes` attribute you can set images for different sizes:
+
+```html
+<template>
+  <v-lazy-image
+    srcset="image.jpg 1x, image_2x.jpg 2x"
```

Comment:
When using the `srcset` attribute is recommended to use also `src` in order to keep the browsers that doesn't support the `srcset` attribute.